MFRC522.cpp

```diff
@@ -13,7 +13,7 @@
 /**
  * Constructor.
  */
-MFRC522::MFRC522() {
```

Comment:
_resetPowerDownPin is type `byte` which is `uint8_t`, so `-1` is not so good idea. 

Code the old code is correct .